feg/gateway/services/testcore/hss/servicers/ai.go

```diff
@@ -72,8 +72,7 @@ func NewAIA(srv *HomeSubscriberServer, msg *diam.Message) (*diam.Message, error)
 		return ConvertAuthErrorToFailureMessage(err, msg, air.SessionID, srv.Config.Server), err
 	}
 
-	const plmnOffsetBytes = 1
-	plmn := air.VisitedPLMNID.Serialize()[plmnOffsetBytes:]
```

Comment:
this changes needs to be reviewed. 

I am not sure why previous code was shifting the plmn by one byle. 

Not removing that code causes PLMN to be just 2 bytes and milenage complains that field should be 3

Reply:
What I understood after checking go-diameter is it takes care of that and provides the value as octet string. but I may be wrong